code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1210,13 +1210,37 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 	switch(H.nutrition)
 		if(NUTRITION_LEVEL_FULL to INFINITY)
 			H.throw_alert("nutrition", /obj/screen/alert/fat)
+			var/obj/item/organ/O = H.getorganslot(ORGAN_SLOT_HEART)
+			O.applyOrganDamage(0.25)
```

Comment:
oh no, I ate a bit too much, I guess I'll die of heart damage now because the chemists didn't make any lipolicide

this also makes potatoes of fatness (potatoes (which are always eaten in a single bite) with high % nutriment and vitamin (and, formerly, sugar) chem production traits) an antag-only gimmick, because now the people who eat them will just straight up die instead of becoming fat.

Reply:
get over it?

Reply:
Oh my god is being fat makes you suffer a heart attack, ahahaha!